sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar.projectVersion=1.0.0-RC8
```

Comment:
Yep, missing `wip` label, sorry

Next step will be github pull request integration & version support